pkg/libvmi/vmi.go

```diff
@@ -158,6 +158,25 @@ func WithLimitCPU(value string) Option {
 	}
 }
 
+func WithNodeSelector(key, value string) Option {
```

Comment:
@0xFelix Can you have a look at this if we already have something?

Reply:
Nothing to add :)